runtime-modules/storage/src/tests/mod.rs

```diff
@@ -6246,15 +6250,18 @@ fn get_upload_costs_succeeds_overflow_sat() {
 #[test]
 fn uploading_objects_with_invalid_cid_length_should_fail() {
     build_test_externalities().execute_with(|| {
-                let starting_block = 1;
+        let starting_block = 1;
         run_to_block(starting_block);
 
-        let initial_balance = 1000;
-        increase_account_balance(&DEFAULT_MEMBER_ACCOUNT_ID, initial_balance);
-
-        let data_object_state_bloat_bond = 10;
+        let data_object_state_bloat_bond = 10u64;
         set_data_object_state_bloat_bond_value(data_object_state_bloat_bond);
 
+        let initial_balance = ExistentialDeposit::get() as u64
+            + data_object_state_bloat_bond
+            + Storage::data_object_per_mega_byte_fee() * DEFAULT_DATA_OBJECTS_SIZE;
```

Comment:
`DEFAULT_DATA_OBJECTS_SIZE` is in bytes, while `data_object_per_mega_byte_fee` is per megabyte